web/src/components/Wizard/Content/Tab.tsx

```diff
@@ -10,7 +10,7 @@ interface IProps {
 
 const Tab = ({index, isActive, step}: IProps) => (
   <S.StepTabContainer $isActive={isActive}>
-    {step.status === 'complete' ? (
+    {step.state === 'completed' ? (
```

Comment:
suggestion: What if we use here a helper method to avoid a hardcoded comparison?
Something like:
```suggestion
    {step.hasCompleted() ? (
```